### DISCUSSION TITLE
connect ETIMEDOUT

### DISCUSSION THREAD

**Discussion by Perceus64** [Budibase/budibase#5773] [Help]
Trying to connect MySQL database to budibase but it keeps giving me the same error "connect ETIMEDOUT".

**Comment by melohagan** [comment]
Hi @Perceus64 

Could you double check your config is correct?

Is it a local instance you are trying to connect to? You may need to use `host.docker.internal` instead of localhost: https://docs.budibase.com/docs/build-a-crud-app-sql#connecting-to-your-sql-tables
